src/java/org/apache/cassandra/net/SensorsCustomParams.java

```diff
@@ -144,6 +148,34 @@ public static <T> void addWriteSensorToResponse(Message.Builder<T> response, Req
                             (sensor) -> SensorsCustomParams.encodeTableInWriteBytesTableParam(sensor.getContext().getTable()));
     }
 
+    /**
+     * A utility method to encode write sensor values in the internode response message. This method will iterate
+     * over all the write sensors in the {@link RequestSensors} and encode the sensor values in the internode response message.
+     */
+    public static <T> void addWriteSensorToResponse(Message.Builder<T> response, RequestSensors sensors)
+    {
+        for (Sensor sensor : sensors.getSensors(Type.WRITE_BYTES))
+        {
+            addSensorToResponse(response, sensor,
+                                (s) -> SensorsCustomParams.encodeTableInWriteBytesRequestParam(sensor.getContext().getTable()),
+                                (s) -> SensorsCustomParams.encodeTableInWriteBytesTableParam(sensor.getContext().getTable()));
+        }
+    }
+
+    /**
+     * A utility method to encode index write sensor values in the internode response message. This method will iterate
+     * over all the write sensors in the {@link RequestSensors} and encode the sensor values in the internode response message.
+     */
+    public static <T> void addIndexWriteSensorToResponse(Message.Builder<T> response, RequestSensors sensors)
```

Comment:
Nit: consider renaming to `addIndexWriteBytes`

Reply:
Fixed as part of the refactoring [commit](https://github.com/datastax/cassandra/pull/1152/commits/910f3a6e134f1601cae981d7f5787bd2d760c3fa)